src/gui/guiFormSpecMenu.cpp

```diff
@@ -556,23 +559,96 @@ void GUIFormSpecMenu::parseScrollBar(parserData* data, const std::string &elemen
 
 		spec.ftype = f_ScrollBar;
 		spec.send  = true;
-		gui::IGUIScrollBar* e =
-				Environment->addScrollBar(is_horizontal,rect,this,spec.fid);
+		GUIScrollBar *e = new GUIScrollBar(Environment, this, spec.fid, rect,
```

Comment:
Needs `e->drop()`.

Reply:
It is already dropped, see https://github.com/minetest/minetest/pull/8530/files#diff-074850a88796e13dad12dd7ae1a8d9f7R126 and https://github.com/minetest/minetest/pull/8530/files#diff-074850a88796e13dad12dd7ae1a8d9f7R2465.

Reply:
Thanks. Although it's currently inconsistent this should work as intended.